jsonb/src/main/java/io/avaje/jsonb/spi/JsonbExtension.java

```diff
@@ -1,11 +1,14 @@
 package io.avaje.jsonb.spi;
 
+import io.avaje.spi.Service;
+
 /**
  * Superclass for all Jsonb SPI classes.
  *
  * @see JsonbComponent
  * @see GeneratedComponent
  * @see AdapterFactory
  */
+@Service
```

Comment:
Technically @Service is not required here. I think it's clearer if it isn't here. I'll remove it.